core/src/components/item-option/item-option.ios.vars.scss

```diff
@@ -4,7 +4,7 @@
 // --------------------------------------------------
 
 /// @prop - Font size of the item option button
-$item-option-button-ios-font-size:          16px !default;
+$item-option-button-ios-font-size:          dynamic-font-clamp(1, 16px, 1.6) !default;
```

Comment:
The iOS native item option uses a `font-size` of `15px` so I used that base to calculate how much the `min` and `max` font size should be. If we update the `font-size` in the future, the min and max ratios do not need to change.